src/modules/75-cf/components/FlagOptionsMenuButton/FlagOptionsMenuButton.tsx

```diff
@@ -92,36 +99,82 @@ const FlagOptionsMenuButton: FC<FlagOptionsMenuButtonProps> = ({
   }
 
   const menuItems = useMemo(() => {
-    const opts = [
-      {
-        icon: 'edit' as IconName,
-        text: getString('edit'),
-        onClick: gotoDetailPage,
-        permission: {
-          resource: { resourceType: ResourceType.ENVIRONMENT, resourceIdentifier: environment },
-          permission: PermissionIdentifier.EDIT_FF_FEATUREFLAG
-        },
-        ...planEnforcementProps
-      },
-      {
-        icon: (FFM_7921_ARCHIVING_FEATURE_FLAGS ? 'archive' : 'trash') as IconName,
-        text: FFM_7921_ARCHIVING_FEATURE_FLAGS ? getString('archive') : getString('delete'),
-        onClick: FFM_7921_ARCHIVING_FEATURE_FLAGS ? openDialog : confirmDeleteFlag,
-        permission: {
-          resource: { resourceType: ResourceType.FEATUREFLAG },
-          permission: PermissionIdentifier.DELETE_FF_FEATUREFLAG
+    let optionItems = [] as RbacMenuItemProps[]
```

Comment:
If you do want to keep this, then type the variable, not the value
```suggestion
    let optionItems: RbacMenuItemProps[] = []
```

Reply:
I had to add that array bc TS was giving off that the contents of the array couldn't be applied to `RbacMenuItemProps`